frontend/src/app/auth/providers/auth/auth.service.ts

```diff
@@ -5,14 +5,20 @@ import { Router } from '@angular/router';
 import { JwtHelperService } from '@auth0/angular-jwt';
 import { User } from '../../../interfaces';
 
+import { LoadingComponent } from "../../../general/components/loading/loading.component";
+import { LoadingController } from "@ionic/angular";
+
 export function tokenGetter() {
   return localStorage.getItem('token');
 }
 
+
 @Injectable({
   providedIn: 'root'
 })
 export class AuthService {
+  loadingController : LoadingComponent = new LoadingComponent(new LoadingController());
```

Comment:
I don't think we should create components like that. @kemaltulum could you check this out?

Further notes, I don't see where you utilize the `LoadingComponent`, you only use the `LoadingController`. The component itself is unnecessary & redundant.